shell/browser/native_window_views.cc

```diff
@@ -630,7 +630,7 @@ bool NativeWindowViews::IsMaximized() {
     return true;
   } else {
 #if BUILDFLAG(IS_WIN)
-    if (transparent()) {
+    if (transparent() && !widget()->IsMinimized()) {
```

Comment:
Nit, but maybe we want to use just `!IsMinimized()` here, in case `NativeWindowViews::IsMinimized` is changed in the future to account for other quirks regarding transparent windows.

Reply:
done!